lib/explorer/data_frame.ex

```diff
@@ -4205,6 +4205,77 @@ defmodule Explorer.DataFrame do
     sample(df, 1.0, seed: opts[:seed], shuffle: true)
   end
 
+  @doc """
+  Transpose a DataFrame.
+
+  > #### Warning {: .warning}
+  >
+  > This is an expensive operation since data is stored in a columnar format.
+
+  ## Options
+
+    * `:include_header` - if `true` column names are added as first column or header column. (default: `false`)
+    * `:header_name` - when `include_header` is `true`, sets name of the header column. (default: `column`)
```

Comment:
We don't need both options. We can have a `:header`option. It may be false (default), true (default to column) or a string with the header name.

Reply:
should `:column_names` be `columns` ?  I see it as a common parameter for many DF functions ?

Reply:
i have changed `:column_names` to `column`. I can revert it..